Source/Scene/Cesium3DTileset.js

```diff
@@ -1223,7 +1239,10 @@ define([
 
         var removeFunction = removeFromProcessingQueue(tileset, tile);
         tile.contentReadyToProcessPromise.then(addToProcessingQueue(tileset, tile));
-        tile.contentReadyPromise.then(removeFunction).otherwise(removeFunction);
+        tile.contentReadyPromise.then(function() {
+            removeFunction();
+            tileset.tileLoad.raiseEvent(tile);
+        }).otherwise(removeFunction);
```

Comment:
Since it looks like this can fail, would it be worth passing an error to the event?